universe/Species.cpp

```diff
@@ -146,6 +146,25 @@ std::string Species::Dump() const {
     return retval;
 }
 
+std::string Species::GameplayDescription() const
+{
+    std::string result(UserString(m_gameplay_description));
+
+    for (std::vector<boost::shared_ptr<Effect::EffectsGroup> >::const_iterator it = m_effects.begin();
+         m_effects.end() != it; ++it)
+    {
+        if (it == m_effects.begin())
```

Comment:
Could there be a layout quirk with this if the first effectsgroup has an empty string? Or will it be fine regardless?